### PR TITLE
fix(core/dfn-finder): dont export local-lt things

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: src/core/dfn-finder.js [hunk 1]
```diff
@@ -71,13 +107,15 @@ function findOperationDfn(defn, parent, name) {
 
 /**
  * @param {HTMLElement} dfn
- * @param {string[]} names
+ * @param {Object} names
```
Comment (sidvishnoi) [inline src/core/dfn-finder.js]:
```suggestion
 * @param {Record<"local" | "exportable", string[]>} names
```

File: src/core/dfn-finder.js [hunk 2]
```diff
@@ -40,21 +40,57 @@ export function findDfn(defn, name, { parent = "" } = {}) {
...
     case "attribute":
-      return [asQualifiedName, name];
+      return {
+        local: [asQualifiedName],
+        exportable: [name],
+      };
   }
 }
 
+function deriveMethodWithArgs(operationName, argsAst) {
```
Comment (sidvishnoi) [inline src/core/dfn-finder.js]: I think a comment to explain what this function does might be better. It's not clear from function name

File: src/core/dfn-finder.js [hunk 3]
```diff
@@ -123,50 +161,54 @@ function findNormalDfn(defn, parent, ...names) {
...
+    !dfnElem.querySelector("code") &&
+    !dfnElem.closest("code") &&
+    dfnElem.children
+  ) {
+    wrapInner(dfnElem, dfnElem.ownerDocument.createElement("code"));
   }
 
-  // Add data-lt values and register them
-  switch (defn.type) {
+  // Add data-lt and data-lt values and register them
```
Comment (sidvishnoi) [inline src/core/dfn-finder.js]: Typo: `data-local-lt`

**Comment by sidvishnoi** [comment]
Didn't see some tests were failing. But changes are good anyway

File: src/core/dfn-finder.js (hunk 2 above)
Comment (marcoscaceres) [reply src/core/dfn-finder.js]: yeah, you are right ... will both rename and document.

**Pull request merged**